apps/lib/app_rand.c

```diff
@@ -82,7 +86,7 @@ int opt_rand(int opt)
     case OPT_R__LAST:
         break;
     case OPT_R_RAND:
-        return loadfiles(opt_arg());
+        load_rand_file = opt_arg();
```

Comment:
With the original code you can load multiple files into the RNG. This functionality should be kept.

Reply:
Loading files into the RNG is only relevant if the (to be) deprecated RAND_METHOD API is used to install an RNG.

I'm not completely convinced that backwards compatibility is necessary.  It would be nice, but I don't see it as a blocker.

Reply:
Really? Don't we have any platforms that do not provide good system entropy? Also in the end the RAND_load_file should be equivalent to call to RAND_add with the data from the file, shouldn't it? So it should have some effect with the default RNG as well. That is for some paranoid folks that want to load some extra entropy data when generating long term keys for example.

Reply:
The behavior of loading multiple files still exists.  The variable name is confusing and I will change it.

Reply:
@richsalz What I meant by multiple files loading was that you could load multiple files not just by -rand <file1>:<file2> but also by using -rand <file1> -rand <file2>. Now this won't be possible as just the file2 will be loaded in this case.

And yes, this behavior is not particularly documented anywhere but ....

Reply:
okay, I added support to allow the -rand flag to be repeated.  new commit pushed.

Reply:
My mistake, we do use the data fed in via RAND_add.  I don't know why I didn't remember this :(